extensions/3rdparty/LyricWiki/Special_WatchlistFeed.body.php

```diff
@@ -241,6 +241,7 @@ function displayWatchlist( $feed, $user ) {
 		$feed->outHeader();
 		$watchlist->prepare();
 		while ( $obj = $watchlist->getItem() ) {
+			$userIp = inet_ntop( $row->rc_ip_bin );
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this unused "$row" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=php%3AS1481)

[project-id]: wikia:app